src/api/ipc.js

```diff
@@ -7,7 +7,7 @@ const reqsSent = new Map();
 
 let ipcEvent = null;
 // Until ipcEvent is ready, we need to keep reqs in a queue
-var pendingReqs = [];
+const pendingReqs = [];
```

Comment:
I don't think this should be const as it needs to be modified afterwards

Reply:
Linter rule is that the value is never reassigned. Looks like `const` is fine here.